src/components/Tooltip/Tooltip.js

```diff
@@ -82,16 +82,23 @@ const Tooltip = (props: Props) => {
 
   const opacityAnim = React.useRef<Animated.Value>(new Animated.Value(0)).current;
   React.useEffect(() => {
-    if (tooltipLayout.height) {
+    let mounted = true;
+    if (tooltipLayout.height && mounted) {
       Animated.timing(opacityAnim, {
         toValue: isVisible ? 1 : 0,
         duration: 200,
         useNativeDriver: true,
-      }).start(() => { if (!isVisible) setVisibilityState(false); });
+      }).start(() => {
+        if (!isVisible) setVisibilityState(false);
+      });
     }
-    if (isVisible) {
+    if (isVisible && mounted) {
       setVisibilityState(true);
     }
+
+    return () => {
+      mounted = false;
```

Comment:
I think I see what you're doing here, but `mounted` isn't a global variable so this wouldn't matter as it just gets set to true the moment the useEffect runs again - if you want to store mounted states, I think its better to use `useState`

Reply:
Here we can't use useState. In this setVisibilityState doing a memory leak because it's call after cleanup. So we just need to stop them after cleaning.

Reply:
Yeah makes sense, like I said above maybe the hook above would be better, or maybe useRef instead?

Reply:
In this `mounted` managed with hooks